core-galleon-pack/src/main/resources/feature_groups/management-legacy-security.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="management-legacy-security" xmlns="urn:jboss:galleon:feature-group:1.0">
```

Comment:
I'm not sure to understand the name `management-legacy-security`. The feature is about adding the management HTTP interface.

Reply:
The http interface is added by the management-interface feature-group. This one is only setting the security-realm on the interface.